api.js

```diff
@@ -1,7 +1,10 @@
 'use strict';
+const isWinOS = /^win/i.test(process.platform);
```

Comment:
This is not the correct way to test Windows.
```suggestion
const isWindows = process.platform === 'win32';
```

Reply:
Changed in *api.js* and *test.js*.